README.md

```diff
@@ -48,10 +48,4 @@ require('gather-ui/dist/styles.css');
 This particular version of the Styleguide depends on the Bootstrap and FontAwesome libraries. These do not ship with the components' CSS and should be included separately wherever they are consumed.
 
 ## Deploying the Library
- You will need to login with an NPM account that has access to the package
-```
-npm version  <major | minor | patch> // We usually go with patch, but minor or major may be required
-git push --tags
-git push origin <branch-name>
-npm publish
-```
\ No newline at end of file
+The library will automatically deploy when a new tag is pushed to the master branch. You can create a tag using the web interface, using the command line with [git tags](https://git-scm.com/book/en/v2/Git-Basics-Tagging), or running `npm version  <major | minor | patch>` (We usually go with patch, but minor or major may be required) will also create a tag for you. Just ensure when pushing you use `git push --tags`.
```

Comment:
I don't think we need to say that we usually patch.

Reply:
Copied from old docs tbh!